tools/cmd.go

```diff
@@ -67,3 +70,33 @@ func mustGetDuration(cmd *cobra.Command, flagName string) time.Duration {
 	}
 	return val
 }
+
+func ResolveManifestFile(input string) (manifestName string, err error) {
+	if input == "" {
+		_, err := os.Stat("substreams.yaml")
+		if err != nil {
+			if errors.Is(err, os.ErrNotExist) {
+				return "", fmt.Errorf("no manifest entered in dir w/o a manifest")
```

Comment:
Don't use text shortcuts like `w/o` for error messages and user facing messages as well as `dir`. There is no good reason to shorten words for user facing elements.